tests/integration/helm_otc_metadata_installation_test.go

```diff
@@ -139,6 +141,30 @@ func Test_Helm_OT_Metadata(t *testing.T) {
 				require.EqualValues(t, 0, daemonsets[0].Status.NumberUnavailable)
 				return ctx
 			}).
+		Assess("metrics are present",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				kubectlOpts := *ctxopts.KubectlOptions(ctx)
+				kubectlOpts.Namespace = "receiver-mock"
+				k8s.WaitUntilServiceAvailable(t, &kubectlOpts, "receiver-mock", int(waitDuration), tickDuration)
+				baseUrl := fmt.Sprintf("http://%s:%d", internal.NodePortHostname, internal.ReceiverMockNodePort)
```

Comment:
Maybe we could think about using tunnel: https://github.com/gruntwork-io/terratest/blob/master/modules/k8s/tunnel_test.go

Reply:
Huh I thought that was a lot more complex for some reason. Let me try this way.